### DISCUSSION TITLE
fractal demo

### DISCUSSION THREAD

**Discussion by kayomarz** [kaveh808/kons-9#214] [General]
Hi everyone,

I recently got down to exploring `kons-9` and made a fractal demo purely for fun. Its rather crude but I plan to improve it.

see [test/demo-fractal-2d-mandelbrot.lisp](https://github.com/kaveh808/kons-9/blob/fractal-demos/test/demo-fractal-2d-mandelbrot.lisp) in branch: [fractal-demos](https://github.com/kaveh808/kons-9/tree/fractal-demos)

**Comment by kaveh808** [comment]
This looks very cool. I'll have a closer look at it soon and run the demo.

**Reply by kayomarz** [reply]
thanks... It is a bit crude but I have ideas for improvements, let me know if you have any suggestions.

**Reply by kaveh808** [reply]
Make a pull request.

**Reply by kayomarz** [reply]
Thank you for suggesting to pull it into main even though the fractal appears pretty crude:)  I will try and improve upon it.

Here's the [pull request](https://github.com/kaveh808/kons-9/pull/218)

**Reply by kaveh808** [reply]
Approved. :)

**Comment by aykaramba** [comment]
This is awesome.  Thanks for the absolutely clear write up.

**Reply by kayomarz** [reply]
thanks :-)

**Comment by ghost** [comment]


**Reply by kayomarz** [reply]
Interesting, thanks for the link.
